README.md

```diff
@@ -10,30 +10,28 @@ Sacrificial VM provides infrastructure for containers.
 
 ### Ports
 
-SSH:
-- Gateway VM SSH Honeypot: `2222`, `22`
-- Gateway VM SSHD: `2333`
-- Other VMs SSHD: `22`
-
-Audit:
+Gateway VM:
```

Comment:
Grouping by VM might be more helpful for looking up opening ports.
The infra diagram can be used to look up available services